app/AccountancyModule/Components/DarkModeToggle.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Components;
+
+use App\AccountancyModule\Components\BaseControl;
+
+final class DarkModeToggle extends BaseControl
+{
+    public function __construct()
+    {
+    }
+
+    // phpcs:disable Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
```

Comment:
proc je tu phpcs disable?

Reply:
Protože jsem to nesmazal, když jsem to kopíroval :D Dík